CONTRIBUTING.md

```diff
@@ -5,10 +5,12 @@ One of the best ways to contribute is to participate in discussions and discuss
 ## General Contributing Guide
 
 1. Unless it is a trivial change, make sure that there is a corresponding issue for your change first. If there is none, create one.
-2. Create a fork in GitHub
-3. Create a branch off the `master` branch. Name it something that that makes sense, such as `issue-123` or `githubhandle-issue`. This makes it easy for everyone to figure out what the branch is used for. It also makes it easier to isolate your change from incoming changes from the origin.
-4. Commit your changes and push your changes to GitHub
-5. Create a pull request against the origin's `master` branch. If you don't know what a pull request is read [this article](https://help.github.com/articles/using-pull-requests).
+2. Create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks) in GitHub.
+3. Create a branch off the `master` branch in your fork. Name it something that that makes sense, such as `issue-123` or `githubhandle-issue`. This makes it easy for everyone to figure out what the branch is used for. It also makes it easier to isolate your change from incoming changes from the upstream repo.
```

Comment:
are we going to have to update this when master dies?

Reply:
Nope. I have a separate change for 2.0-preview.